tools-integration/src/test/java/com/hedera/hashgraph/client/integration/TestUtil.java

```diff
@@ -214,7 +214,15 @@ public static Button findButtonFromPopup(String buttonName) {
 	public static Stage findModalWindow() {
 		// Get a list of windows but ordered from top[0] to bottom[n] ones.
 		// It is needed to get the first found modal window.
-		final List<Window> allWindows = new ArrayList<>(robot.robotContext().getWindowFinder().listWindows());
+		final var windowFinder = robot.robotContext().getWindowFinder();
+		if (windowFinder == null) {
+			return null;
+		}
+		final List<Window> allWindows = new ArrayList<>(windowFinder.listWindows());
+		if (allWindows.isEmpty()) {
+			return null;
```

Comment:
It might be better to return empty ArrayList instead of null

Reply:
Actually I don't return a list, but a stage. In this particular case a null is better than an empty stage.